ios/Runner/Info.plist

```diff
@@ -28,6 +28,17 @@
 	<string>To get location and direction</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>To get location and direction</string>
+	<key>NSMicrophoneUsageDescription</key>
+    <string>not used</string>
```

Comment:
if this is not used, we can remove it

Reply:
Required I think, even if it's not used
https://pub.dev/packages/just_audio#ios